tests/frontDesk/visits.js

```diff
@@ -167,4 +167,19 @@ step("Validate obs <form> on the patient clinical dashboard", async function (fo
     await taikoHelper.repeatUntilNotFound($("#overlay"))
     await taikoHelper.validateFormFromFile(obsFormValues.ObservationFormDetails, obsFormValues.ObservationFormName)
     await click($('.ngdialog-close'))
+});
+
+step("Verify medical test in patient clinical dashboard", async function () {
+    await taikoHelper.repeatUntilNotFound($(".dashboard-section-loader"))
+    var labOrderCount = gauge.dataStore.scenarioStore.get("labOrderCount")
+    console.log("count " + labOrderCount)
+    for (var i = 0; i < labOrderCount; i++) {
+        var labOrderFile = gauge.dataStore.scenarioStore.get("labOrder" + i)
+        var testLabOrder = JSON.parse(fileExtension.parseContent(labOrderFile))
+        for (var j = 0; j < testLabOrder.Result.length; j++) {
+            console.log(testLabOrder.Result[j].label)
+            console.log(testLabOrder.Result[j].value)
+            assert.ok(await text(`${testLabOrder.Result[j].value}`, toRightOf(`${testLabOrder.Result[j].label}`)).exists())
```

Comment:
if possible add `within` to make sure these values are present inside the lab widget.

Reply:
something like below
`assert.ok(await text(medicalPrescriptions.drug_name, within($("#Treatments"))).exists())`